core/src/com/unciv/ui/overviewscreen/ReligionOverviewTable.kt

```diff
@@ -0,0 +1,117 @@
+package com.unciv.ui.overviewscreen
+
+import com.badlogic.gdx.graphics.Camera
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.ui.Button
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.unciv.logic.civilization.CivilizationInfo
+import com.unciv.models.Religion
+import com.unciv.models.ruleset.Belief
+import com.unciv.models.translations.tr
+import com.unciv.ui.utils.*
+import kotlin.math.min
+
+class ReligionOverviewTable(
+    private val viewingPlayer: CivilizationInfo,
+    private val overviewScreen: EmpireOverviewScreen
+): Table() {
+    
+    val gameInfo = viewingPlayer.gameInfo
+    private val topButtons = Table(CameraStageBaseScreen.skin)
+    private val topButtonLabel = "Click an icon to see the stats of this religion".tr().toLabel()
+    private val statsTable = Table(CameraStageBaseScreen.skin)
+    private val beliefsTable = Table(CameraStageBaseScreen.skin)
+    private var selectedReligion: String? = null
+    
+    init {
+        draw(null)
+    }
+    
+    private fun draw(religion: Religion?) {
```

Comment:
Give this function a better name :)
displayReligionInfo, perhaps?

Reply:
update-something is already often used for similar usecases